pkg/agent/openflow/network_policy_test.go

```diff
@@ -1416,3 +1417,70 @@ func Test_featureNetworkPolicy_initFlows(t *testing.T) {
 		})
 	}
 }
+
+func Test_NewDNSPacketInConjunction(t *testing.T) {
```

Comment:
Great test. I assume it will fail without the patch, right?

Reply:
Yes, it is failed before the change.